apparmor.d/groups/kde/konsole

```diff
@@ -0,0 +1,59 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2023 Jeroen Rijken
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{usr/,}bin/konsole
+profile konsole @{exec_path} flags=(attach_disconnected) {
+  include <abstractions/base>
+  include <abstractions/bus-accessibility>
+  include <abstractions/consoles>
+  include <abstractions/dri>
+  include <abstractions/fonts>
+  include <abstractions/freedesktop.org>
+  include <abstractions/kde-strict>
+  include <abstractions/nameservice-strict>
+  include <abstractions/ubuntu-konsole>
+
+  ptrace (read),
+
+  @{exec_path} mr,
+  @{bin}/** rPUx,
+  @{user_bin_dirs} rPUx,
```

Comment:
Is it needed?

Terminal app ususally only need to start a shell unconfined. Like
```
  @{bin}/@{shells}      rUx,
```